starfile/writer.py

```diff
@@ -128,7 +128,7 @@ def write_block(self, df: pd.DataFrame):
         self.buffer.append_to_file_and_clear(self.filename)
 
     def _write_simple_block(self, df: pd.DataFrame):
-        lines = [f'_{column_name}\t\t\t{df[column_name][0]}'
+        lines = [f'_{column_name}\t\t\t{df[column_name].iloc[0]}'
```

Comment:
switched for an explicit `df.iloc` call - more readable/standard way of doing this